pkg/device/nvidia/device.go

```diff
@@ -131,13 +136,11 @@ type FilterDevice struct {
 
 type DevicePluginConfigs struct {
 	Nodeconfig []struct {
-		Name                string        `json:"name"`
-		OperatingMode       string        `json:"operatingmode"`
-		Devicememoryscaling float64       `json:"devicememoryscaling"`
-		Devicecorescaling   float64       `json:"devicecorescaling"`
-		Devicesplitcount    uint          `json:"devicesplitcount"`
-		Migstrategy         string        `json:"migstrategy"`
-		FilterDevice        *FilterDevice `json:"filterdevices"`
+		NvidiaConfig  NodeDefaultConfig `json:"nvidiaconfig"`
```

Comment:
1. Please do not change the current field in this struct, which leads to a backward compatible issue. I think it is better to use `json:,inline` tag to keep it compatible with previous releases. 
2. Please leave a comment here to inform that this config is shared by `NvidiaConfig` struct